### PR TITLE
Chapter1/homework/cooking new

### PR DESCRIPTION
**Pull request by EdvinasA**
I don't know if i did correctly with exceptions but i tried to put in exceptions in methods were they fitted, also created recipe processing from file, you need to enter file location and name if file is in java-homework folder you only need to enter the name of the file, also fixed reading recipes, now no matter the order it will print ingredient first amount second and unit third, haven't implemented if there are missing some of the parts of recipe. also used arrayLists for finding recipe parts.

Implemented tests.

haven't figured out yet how to remove units and put them in enum class with amounts.

### CODE REVIEW

File: src/main/java/CookingConvertor/Amounts.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            case "CUP":
+                return CUP.getNumber();
+            case "FL_OUNCE":
+                return FL_OUNCE.getNumber();
+            case "TBSP":
+                return TBSP.getNumber();
+            default:
+                return 0;
+        }
+    }
```
Comment (Almantask) [inline src/main/java/CookingConvertor/Amounts.java]: No longer needed- enum represents that string.

File: src/main/java/CookingConvertor/Cooking.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+        public static boolean isValid (String input) {
+            for (String value : values) {
+                if (input.toUpperCase().equals(value.toUpperCase())) {
+                    return false;
+                }
+            }
+            return true;
+        }
+    }
```
Comment (Almantask) [inline src/main/java/CookingConvertor/Cooking.java]: The whole class can now be removed, because you use enums

Comment (Almantask) [reply src/main/java/CookingConvertor/Cooking.java]: Enum is always valid

File: tests/CookingConvertor/CookingMeasurementConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package CookingConvertor;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CookingMeasurementConverterTest {
```
Comment (Almantask) [inline tests/CookingConvertor/CookingMeasurementConverterTest.java:6]: Missing many tests

**Review by Almantask — CHANGES_REQUESTED** [review]

File: src/main/java/CookingConvertor/Cooking.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package CookingConvertor;
+
+public class Cooking {
+
+    public static void main(String[] args) {
+        ReceipePrompt prompt = new ReceipePrompt();
+        prompt.run();
+    }
+}
+
```
Comment (EdvinasA) [inline src/main/java/CookingConvertor/Cooking.java:10]: Removed Units class

**Pull request closed without merging**